tests/FakeItEasy.Specs/OrderedCallMatchingSpecs.cs

```diff
@@ -89,6 +96,49 @@ public interface IHaveOneGenericParameter
 "));
         }
 
+        [Scenario]
+        public static void WithRepeatConstraint(
+            IHaveNoGenericParameters fake,
+            CallOrderingContext callOrderingContext,
+            Exception exception)
+        {
+            "Given a Fake"
+                .x(() => fake = A.Fake<IHaveNoGenericParameters>());
+
+            "And a call on the Fake, passing parameter 1"
+                .x(() => fake.Bar(1));
+
+            "And a call on the Fake, passing parameter 2"
+                .x(() => fake.Bar(2));
+
+            "And a call on the Fake, passing parameter 1"
+                .x(() => fake.Bar(1));
+
+            "And a call-ordering context"
+                .x(c => callOrderingContext = A.CallOrderingContext().Using(c));
+
+            "When I expect the call with parameter 2 to have been made"
+                .x(() => A.CallTo(() => fake.Bar(2)).MustHaveHappened().InOrder(callOrderingContext));
+
+            "And I expect the call with parameter 1 to have been made next exactly once"
+                .x(() => exception = Record.Exception(() => A.CallTo(() => fake.Bar(1)).MustHaveHappened(Repeated.Exactly.Once).InOrder(callOrderingContext)));
+
+            "Then the assertion should fail"
+                .x(() => exception.Should().NotBeNull("because the assertion should have failed"));
+
+            "And the error should tell us that the calls were not matched in order"

```

Comment:
I am either reading things wrong or it was not fixed in the latest update. (Just making sure I'm understanding things.)